ebau_gwr/linker/filters.py

```diff
@@ -0,0 +1,61 @@
+import json
+
+from django.contrib.postgres.fields.jsonb import KeyTextTransform
+from django_filters import Filter, FilterSet
+from django_filters.constants import EMPTY_VALUES
+from rest_framework.exceptions import ValidationError
+
+from . import models
+
+
+class JSONValueFilter(Filter):
```

Comment:
We're copying that thing everywhere, probably should start to think about putting it in it's own repo sometime soon :)

... along with the test_api stuff as well